vnext/Shared/DevSupportManager.cpp

```diff
@@ -232,6 +235,28 @@ void DevSupportManager::StopPollingLiveReload() {
   m_cancellation_token = true;
 }
 
+void DevSupportManager::startInspector(const std::string &packagerHost, const uint16_t packagerPort) {
```

Comment:
>packagerHost [](start = 58, length = 12)

There will be a compilation error if these variables are not used. Please add [[maybe_unused]] attributes for them.

Reply:
Done .. thanks !